api.js

```diff
@@ -116,6 +116,19 @@ Api.prototype._run = function (files, _options) {
 	self.precompiler = new CachingPrecompiler(cacheDir, self.options.babelConfig);
 	self.fileCount = files.length;
 
+	var overwatch;
+	if (this.options.concurrency > 0) {
```

Comment:
But if I add a `concurrency` option to `package.json`, I want to be able to re-enable `.only` behavior. By doing 

`ava [files ...]`

Where the number of files is less than concurrency.